lib/interactive_viewer.ml

```diff
@@ -4,11 +4,19 @@ open Lwd_infix
 
 let operation_info z_patches : ui Lwd.t =
   let$ z = Lwd.get z_patches in
+  let p = Zipper.get_focus z in
+  let num_hunks = List.length p.Patch.hunks in
+  let hunk_text = match num_hunks with
+    | 1 -> "1 hunk"
+    | n when n > 1 -> Printf.sprintf "%d hunks" n
+    | _ -> "0 hunks"
```

Comment:
Those two cases could be simplified into one!

Reply:
Okay!
I have combined the cases to improve code readability and clarity.
@panglesd

Reply:
@panglesd and @Julow 
Could you kindly take a look at this implementation I made after review?

Thanks.